### PR TITLE
Merged interstitial page/redirect domain improvements

### PR DESCRIPTION
**Pull request by wayne-chen**
- unified experience of #14092 and #14089 

Test Plan:
Interstitial page section
https://highfidelity.testrail.net/index.php?/suites/view/1357&group_by=cases:section_id&group_order=asc
Note: This is disabled by default, use `Window.interstitialModeEnabled = true` then ctrl-r to enable it.

This PR will address this issue [18557](https://highfidelity.manuscript.com/f/cases/18557)

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/13381/HighFidelity-Beta-PR14094-813defe.apk

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/13381/HighFidelity-Beta-PR14094-813defe.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/13381/HighFidelity-Beta-Interface-PR14094-813defe.exe

**Comment by JeffClinton** [comment]
Build this please

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/13394/HighFidelity-Beta-PR14094-813defe.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/13395/HighFidelity-Beta-PR14094-747aab2.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/13395/HighFidelity-Beta-Interface-PR14094-747aab2.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/13395/HighFidelity-Beta-PR14094-747aab2.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/13395/HighFidelity-Beta-Interface-PR14094-747aab2.exe

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/13401/HighFidelity-Beta-PR14094-9079c8a.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/13401/HighFidelity-Beta-PR14094-9079c8a.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/13401/HighFidelity-Beta-Interface-PR14094-9079c8a.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/13401/HighFidelity-Beta-PR14094-9079c8a.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/13401/HighFidelity-Beta-Interface-PR14094-9079c8a.exe

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/13421/HighFidelity-Beta-PR14094-8b29f7c.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/13421/HighFidelity-Beta-Interface-PR14094-8b29f7c.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/13421/HighFidelity-Beta-PR14094-8b29f7c.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/13421/HighFidelity-Beta-Interface-PR14094-8b29f7c.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/13427/HighFidelity-Beta-PR14094-5d27cdd.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/13427/HighFidelity-Beta-Interface-PR14094-5d27cdd.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/13427/HighFidelity-Beta-PR14094-5d27cdd.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/13427/HighFidelity-Beta-Interface-PR14094-5d27cdd.dmg

**Comment by wayne-chen** [comment]
build this please

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/13437/HighFidelity-Beta-PR14094-5d27cdd.apk

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/13437/HighFidelity-Beta-PR14094-5d27cdd.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/13437/HighFidelity-Beta-Interface-PR14094-5d27cdd.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/13437/HighFidelity-Beta-PR14094-5d27cdd.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/13437/HighFidelity-Beta-Interface-PR14094-5d27cdd.dmg

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/13442/HighFidelity-Beta-PR14094-5c9094d.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/13442/HighFidelity-Beta-PR14094-5c9094d.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/13442/HighFidelity-Beta-Interface-PR14094-5c9094d.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/13442/HighFidelity-Beta-PR14094-5c9094d.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/13442/HighFidelity-Beta-Interface-PR14094-5c9094d.exe

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/13449/HighFidelity-Beta-PR14094-a8b86bd.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/13449/HighFidelity-Beta-PR14094-a8b86bd.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/13449/HighFidelity-Beta-Interface-PR14094-a8b86bd.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/13449/HighFidelity-Beta-PR14094-a8b86bd.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/13449/HighFidelity-Beta-Interface-PR14094-a8b86bd.exe

**Comment by wayne-chen** [comment]
build this please

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/13498/HighFidelity-Beta-PR14094-a8b86bd.apk

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/13501/HighFidelity-Beta-PR14094-eab5763.apk

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/13505/HighFidelity-Beta-PR14094-5f05be5.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/13505/HighFidelity-Beta-PR14094-5f05be5.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/13505/HighFidelity-Beta-Interface-PR14094-5f05be5.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/13505/HighFidelity-Beta-PR14094-5f05be5.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/13505/HighFidelity-Beta-Interface-PR14094-5f05be5.exe

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/13534/HighFidelity-Beta-PR14094-8f36bc0.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/13536/HighFidelity-Beta-PR14094-36dbcd4.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/13536/HighFidelity-Beta-Interface-PR14094-36dbcd4.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/13536/HighFidelity-Beta-PR14094-36dbcd4.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/13536/HighFidelity-Beta-Interface-PR14094-36dbcd4.exe

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/13596/HighFidelity-Beta-PR14094-2d85f88.apk

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/13596/HighFidelity-Beta-PR14094-2d85f88.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/13596/HighFidelity-Beta-Interface-PR14094-2d85f88.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/13606/HighFidelity-Beta-PR14094-efeb598.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/13606/HighFidelity-Beta-Interface-PR14094-efeb598.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/13606/HighFidelity-Beta-PR14094-efeb598.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/13606/HighFidelity-Beta-Interface-PR14094-efeb598.exe

**Comment by qligmaloney** [comment]
Build this please

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/13609/HighFidelity-Beta-PR14094-efeb598.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/13609/HighFidelity-Beta-Interface-PR14094-efeb598.dmg

**Comment by JeffClinton** [comment]
QA Ready tag removed since another fix is coming.

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/13619/HighFidelity-Beta-PR14094-f875bfd.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/13619/HighFidelity-Beta-PR14094-f875bfd.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/13619/HighFidelity-Beta-Interface-PR14094-f875bfd.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/13619/HighFidelity-Beta-PR14094-f875bfd.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/13619/HighFidelity-Beta-Interface-PR14094-f875bfd.exe

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/13634/HighFidelity-Beta-PR14094-22a9030.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/13634/HighFidelity-Beta-PR14094-22a9030.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/13634/HighFidelity-Beta-Interface-PR14094-22a9030.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/13634/HighFidelity-Beta-PR14094-22a9030.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/13634/HighFidelity-Beta-Interface-PR14094-22a9030.exe

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/13640/HighFidelity-Beta-PR14094-0a7102b.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/13640/HighFidelity-Beta-PR14094-0a7102b.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/13640/HighFidelity-Beta-Interface-PR14094-0a7102b.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/13640/HighFidelity-Beta-PR14094-0a7102b.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/13640/HighFidelity-Beta-Interface-PR14094-0a7102b.exe

**Comment by qlistephens95** [comment]
Currently Testing

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/13649/HighFidelity-Beta-PR14094-c785efe.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/13649/HighFidelity-Beta-PR14094-c785efe.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/13649/HighFidelity-Beta-Interface-PR14094-c785efe.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/13649/HighFidelity-Beta-PR14094-c785efe.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/13649/HighFidelity-Beta-Interface-PR14094-c785efe.exe

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/13660/HighFidelity-Beta-PR14094-2e3566b.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/13660/HighFidelity-Beta-PR14094-2e3566b.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/13660/HighFidelity-Beta-Interface-PR14094-2e3566b.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/13660/HighFidelity-Beta-PR14094-2e3566b.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/13660/HighFidelity-Beta-Interface-PR14094-2e3566b.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/13667/HighFidelity-Beta-PR14094-89e2cf7.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/13667/HighFidelity-Beta-Interface-PR14094-89e2cf7.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/13667/HighFidelity-Beta-PR14094-89e2cf7.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/13667/HighFidelity-Beta-Interface-PR14094-89e2cf7.exe

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/13671/HighFidelity-Beta-PR14094-fc17e0e.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/13671/HighFidelity-Beta-PR14094-fc17e0e.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/13671/HighFidelity-Beta-Interface-PR14094-fc17e0e.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/13671/HighFidelity-Beta-PR14094-fc17e0e.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/13671/HighFidelity-Beta-Interface-PR14094-fc17e0e.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/13673/HighFidelity-Beta-PR14094-8abbd51.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/13673/HighFidelity-Beta-Interface-PR14094-8abbd51.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/13673/HighFidelity-Beta-PR14094-8abbd51.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/13673/HighFidelity-Beta-Interface-PR14094-8abbd51.exe

**Comment by qliejahner** [comment]
Currently testing

**Comment by qliejahner** [comment]
FAIL @conklin94122 

In VR mode, when clicking "Go To TheSpot" while loading another domain, TheSpot may never stop loading. I was stuck for more than 5 minutes, unable to move, unable to bring up the tablet, and, since I'm already going to TheSpot, unable to abort the load. As a VR user, it was a dead end. This happened 3 out of 10 times.

image

While stuck in this state, another tester could see me. I was also able to teleport, and see the teleport beam colliding with the domain. The other tester could see my frozen body jumping around.

screen shot 2018-10-05 at 3 04 26 pm
screen shot 2018-10-05 at 3 05 01 pm


When a connection timed out, the connection was automatically retried, causing an indefinite loop between the void, 404 and loading screens. There's an opportunity to back out of this, but it's a bad user experience.

desktop 2018 10 05 - 14 50 32 01


On a less critical note: "share your snapshots with the everyone"

image

[hifi-log.zip](https://github.com/highfidelity/hifi/files/2452250/hifi-log.zip)

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/13687/HighFidelity-Beta-PR14094-f758438.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/13687/HighFidelity-Beta-PR14094-f758438.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/13687/HighFidelity-Beta-Interface-PR14094-f758438.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/13687/HighFidelity-Beta-PR14094-f758438.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/13687/HighFidelity-Beta-Interface-PR14094-f758438.exe

**Comment by qliejahner** [comment]
Currently testing

**Comment by qliejahner** [comment]
PASS @conklin94122 

Can no longer reproduce indefinite load going into TheSpot. Some loads took a bit longer than usual, and initially appeared to be stuck like before, but all eventually resolved.

There is a rare bug in which the load progress is skewed and does not fit inside the bar. This happened on each screen on my first session. After re-launching interface, it did not occur again, even after clean installs. It also happened on exactly one loading screen on Mac.

However, this is an oddly rare visual defect which I don't consider severe enough to fail the PR.

image

File: interface/src/octree/SafeLanding.cpp [hunk 1]
```diff
@@ -130,11 +142,16 @@ bool SafeLanding::isSequenceNumbersComplete() {
...
         auto startIter = _sequenceNumbers.find(_initialStart);
         auto endIter = _sequenceNumbers.find(_initialEnd - 1);
+
+        bool missingSequenceNumbers = qApp->isMissingSequenceNumbers();
         if (sequenceSize == 0 ||
             (startIter != _sequenceNumbers.end()
             && endIter != _sequenceNumbers.end()
-            && distance(startIter, endIter) == sequenceSize - 1) ) {
-            _trackingEntities = false; // Don't track anything else that comes in.
+             && ((distance(startIter, endIter) == sequenceSize - 1) || !missingSequenceNumbers))) {
```
Comment (sethalves) [inline interface/src/octree/SafeLanding.cpp]: strange indentation here

File: libraries/entities-renderer/src/RenderableModelEntityItem.cpp [hunk 2]
```diff
@@ -315,6 +315,14 @@ void RenderableModelEntityItem::getCollisionGeometryResource() {
     _compoundShapeResource = DependencyManager::get<ModelCache>()->getCollisionGeometryResource(hullURL);
 }
 
+bool RenderableModelEntityItem::computeShapeFailedToLoad() {
+    if (!_compoundShapeResource) {
+        getCollisionGeometryResource();
+    }
+
+    return (_compoundShapeResource && _compoundShapeResource->isFailed());
```
Comment (sethalves) [inline libraries/entities-renderer/src/RenderableModelEntityItem.cpp]: something about this function is off.

**Pull request merged**